### PR TITLE
DEVOPS-2915 add postgresdump status

### PR DESCRIPTION
**Pull request by kunalp-gohire**
add promt select, add postgresdump status

### CODE REVIEW

**Review by vivekjainx86 — APPROVED** [review]
looks good to me

File: pkg/cmd/status.go [hunk 1]
```diff
@@ -65,81 +65,125 @@ func getData(objType string, context string, namespace string, tenant string) (s
...
-					return err
-				}
-				p.Increment()
 				_, _ = p.Stop()
 				_ = area.Stop()
-
 			}
 		} else {
-			pterm.Success.Printf(sData + "\n" + dData + "\n")
+			if statusType == "summonplatform" {
```
Comment (shubhindia) [inline pkg/cmd/status.go:182]: This will never execute because we don't have status type as summonplatform anymore

File: pkg/cmd/status.go [hunk 2]
```diff
@@ -65,81 +66,130 @@ func getData(objType string, context string, namespace string, tenant string) (s
...
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-
 		utils.CheckVPN()
 		utils.CheckKubectl()
 		return nil
 	},
-	RunE: func(_ *cobra.Command, args []string) error {
+	RunE: func(cmd *cobra.Command, args []string) error {
+		statusTypes := []string{"Summon Platform Status", "DB Backup Status"}
```
Comment (shubhindia) [inline pkg/cmd/status.go:96]: No need to have `Status` here as the command itself is for status. only summon platform and db backup is enough

**Review by shubhindia — APPROVED** [review]
looks good to me

**Review by shubhindia — APPROVED** [review]

**Review by avdhoot — APPROVED** [review]

**Pull request merged**